publiccode.yml

```diff
@@ -0,0 +1,100 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/publiccodeyml/publiccode.yml
+
+publiccodeYmlVersion: "0.3"
+
+# Dit veld bevat de korte naam van het project. Kies hier de naam waaronder het
```

Comment:
The rest of the repo contains code and comments only written in English, should we remove these comments or rewrite them in English to keep the codebase as accessible as possible to the community?

Reply:
I guess you're right. The file is intended more for software ingestion and I just left the template comments in there for reference. I'll remove them since people can just read the spec.